account_invoice_inter_company/models/account_move.py

```diff
@@ -52,6 +56,19 @@ def action_post(self):
             )._inter_company_create_invoice(dest_company)
         return res
 
+    def _attach_original_pdf_report(self):
+        pdf = self.env.ref("account.account_invoices")._render_qweb_pdf([self.id])[0]
```

Comment:
I thinks there is something wrong here.
We can have several case of invoice 
Out and in invoice.

- Example 1: company A create an out invoice company b. We have to attach the pdf of company A into company B

- Example 2: (more rare case) company B generate an in invoice and this generate the out invoice in company A. We have to attach the pdf of company A into company B

In both case we attach the pdf of company A (out invoice) into company B (in invoice)
But the value of auto_invoice_id is not the same!

Reply:
Also take in account refund case

Reply:
As you suggested, the module take into account, both cases.